src/main/java/sirius/biz/storage/layer2/L3Uplink.java

```diff
@@ -118,18 +118,16 @@ Blob createAsBlob() {
          * @param name            the name of the placeholder
          * @return a placeholder representing a non existent file or directory
          */
-        protected Optional<VirtualFile> createPlaceholder(@Nullable Directory parentDirectory,
-                                                          VirtualFile parent,
-                                                          String name) {
+        protected VirtualFile createPlaceholder(@Nullable Directory parentDirectory, VirtualFile parent, String name) {
```

Comment:
Could we annotate the public and protected functions with Nullable and Nonnull when we remove the usage of Optional? This would be helpful for callers of the API and ovveriders of the protected methods by an extending API.

Also its generally good codestyle and indicates that thought was put into wether or not the method can return null or not.